src/core/context/legacy-slashauth.tsx

```diff
@@ -270,6 +271,7 @@ const emptyContext = {
   getIdTokenClaims: uninitializedStub,
   checkSession: uninitializedStub,
   mountSignIn: uninitializedStub,
+  goToAccountPage: uninitializedStub,
```

Comment:
This is a bit strange because we aren't going to redirect a browser. This will either render a modal or show a drop in component, not a new route. This shouldn't be exposed here.

Reply:
updated